### ISSUE TITLE
Clean Loop list A11y

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4890]
There are places where lists of items are only shown visually, and not included as lists in the HTML (e.g. separated by `<br/>` or commas).  These should be marked up as lists.  

Lists are important syntax where loops have been used.  Visually a user may know 'this is a handful of items' but for a screen reader user, using list syntax tells them how many items, and where they are in the list of items, as well as allowing them to skip quickly past the list where an unknown length of comma separated items would only provide that visually.

So:
- search for all places in the Clean theme where loops are used and ensure  it is marked up as a list. 

Note: to keep the visual consistent, there are already styles from the other list A11y work for inline lists, and those separated by specific characters (e.g. a bar) (see `clean.css`).

Consider: whether templates for common list types could be useful.
